data/human/human missions.txt

```diff
@@ -6891,3 +6891,370 @@ mission "Returning to Paradise"
 			action
 				payment 120000
 			`You're not sure if you've ever met a more pretentious and entitled family. The girls fought non-stop, but fortunately, the family was satisfied with the luxury accommodations on your ship. Once you land on <planet>, the lady gives you <payment> and the family leave without a word of thanks; they never even tell you their names. Nonetheless, you're happy to be rid of them.`
+
+
+
+mission "Adelita 1"
+	name "Exhibition installations to <planet>"
+	description `Transport <tons> of exhibition installations to <destination> for the Adelita collective. Payment is <payment>.`
+	minor
+	cargo "exhibition installations" 5
+	source "Follower"
+	destination "Moonshake"
+	to offer
+		or
+			random < 10
+			and
+				random < 60
+				has "Expedition to Hope 2: done"
+		"combat rating" > 40
+
+	on offer
+		conversation
+			branch "has visited"
+				has "visited planet: Hope"
+			`Walking on the edges of <origin>'s spaceport district, you notice signage advertising an exhibition based on the planet Hope and the disaster and evacuation that happened on it in 2994; you vaguely remember it as being a planet in a system southwest of New Boston. But what makes it stand out from the typical advertisements that you see on notice boards and interactive screens is its colorful language - with phrases like "Considered Offensive by Paradisians," "Why We Can't Give Up Hope," and "Evicted by the Paradise Planets."`
+				goto directions
+
+			label "has visited"
+			`Walking on the edges of <origin>'s spaceport district, you notice signage advertising an exhibition based on the planet Hope and the disaster and evacuation that happened on it in 2994; you remember having been there once before on an excursion. But what makes it stand out from the typical advertisements that you see on notice boards and interactive screens is its colorful language - with phrases like "Considered Offensive by Paradisians," "Why We Can't Give Up Hope," and "Evicted by the Paradise Planets."`
+
+			label directions
+			`	At the bottom of each sign are the directions to get to the exhibition. It appears to be located in a warehouse yard nearby, where the spaceport district borders the city that surrounds it.`
+			choice
+				`	(Check it out.)`
+				`	(Not interested.)`
+					decline
+			`	Following the signage you notice a significant group of people gathered in front in front of a large warehouse that stands out amongst the rest of the warehouses in the yard. As you wait in the queue a young woman lines up behind you with a readily apparent exuberance and asks you, "I'd like to know, out of interest, how much you know about Hope and its history?"`
+			choice
+				`	"Not much at all."`
+				`	"I come from the Dirt Belt, but I really don't know too much about what happened on Hope."`
+				`	"I've been there myself and know what happened, but other than that I don't know."`
+					goto "been to hope"
+					to display
+						has "visited planet: Hope"
+						or
```

Comment:
If you've been to Hope but not done either of those two missions, you know someone who's been affected and have been told why.

Reply:
Are the options not intended to be mutually exclusive? I am getting both of them when I have done only one of the missions.
<img width="577" alt="image" src="https://github.com/endless-sky/endless-sky/assets/98854112/b5a4f35f-d6e2-44e3-8385-7d81d2db4f94">
If you want them to be mutually exclusive (either you know someone or you don't), then you need an `and` instead of an `or`.